coralproject/talk
# Adding traditional Chinese locales for Coral talk

## What does this PR do?
Created a traditional Chinese locales, so that talk can support Chinese.

## How do I test this PR?
- set TALK_DEFAULT_LANG to 'zh_TW'
- run the talk instance
- see if everything in proper traditional Chinese
